docs/content.zh/docs/dev/table/sql/create.md

```diff
@@ -161,8 +161,8 @@ CREATE TABLE [IF NOT EXISTS] [catalog_name.][db_name.]table_name
     [ <table_constraint> ][ , ...n]
   )
   [COMMENT table_comment]
-  [PARTITIONED BY (partition_column_name1, partition_column_name2, ...)]
   [ <distribution> ]
+  [PARTITIONED BY (partition_column_name1, partition_column_name2, ...)]
```

Comment:
to make it matching actual implementation
https://github.com/apache/flink/blob/5db1ffac6eb4f5e291b0274261be31d72196087d/flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl#L1579-L1587